Snippets/Snippets_6/UpgradeGuides/5to6/UpgradeMessageHandler.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Snippets6.Handlers
+{
+    using System.Threading.Tasks;
+    using NServiceBus;
+
+    #region 5to6-messagehandler
+    public class UpgradeMySynchronousHandler : IHandleMessages<MyMessage>
+    {
+        public Task Handle(MyMessage message, IMessageHandlerContext context)
+        {
+            //use this example for synchronous code
```

Comment:
perhaps

    // when no asynchronous code is executed in a handler Task.FromResult(0) can be returned